src/test/java/com/imgix/test/TestSrcSet.java

```diff
@@ -561,4 +561,88 @@ public void testHeightAndAspectRatioIncludesDPRParam() {
             assert(src.contains(String.format("dpr=%s", i+1)));
         }
     }
+
+    @Test
```

Comment:
Mind adding tests for the following?
- If `begin` and `end` are equal
- `createSrcSet` respects an overriding `q` when generating a DPR srcset

Reply:
Good call! Yes I will!